README.md

```diff
@@ -1,5 +1,31 @@
 
-# Bughunter
+# Tantalus (Bughunter 2.0)
+
+## Requirements
+
+To run a local development environment, you must have [Docker](https://www.docker.com/get-started) installed and its daemon running.
+
+Currently, to access Tantalus, you need an LDAP account, as well as access to the bughunter LDAP group. (Soon to be changed to the tantalus LDAP group)
+
+You also need to be connected to [Mozilla's VPN](https://mana.mozilla.org/wiki/display/SD/VPN).
+
+## Usage
+
+To run the web app and database containers, type the following command into your terminal:
+
+```bash
+docker-compose build && docker-compose up
```

Comment:
Nit: on the first run of `docker-compose up`, when a bughunter container has not yet been built, the build command happens automatically. `docker-compose build` only needs to be run if the `Dockerfile` referenced by the `docker-compose.yml` file has changed.